### ISSUE TITLE
Consider URL for filtering

### ISSUE THREAD

**Issue by moltar** [deanishe/alfred-firefox#29]
It appears that URL is not considered when filtering tabs. I'm using a productivity app (ClickUp) that does not have the app name in the `<title>`, and thus it makes it difficult to search for it.

It'd be ideal if URL was searchable too.

Thanks!

**Comment by KingMob** [comment]
Yes, it's especially tricky with tabs where the title changes, like GMail. The URL should be at least as, if not more, important.
